pkg/server/apis/v1alpha1/types.go

```diff
@@ -248,3 +248,40 @@ type StorageUsage struct {
 type StorageCleanup struct {
 	Paths []string `json:"paths"`
 }
+
+// ExecutionContextSpec describes the execution context
+type ExecutionContextSpec struct {
+	Cluster     string `json:"cluster"`
+	Namespace   string `json:"namespace"`
+	Integration string `json:"integration"`
```

Comment:
Do we need `integration` ?

Reply:
I will add a `Closed` phase for execution context.
If users want to use a closed execution context, they need to open the related cluster integration firstly. 
So I think the `integration` field is useful.

Reply:
I'd prefer to remove it.